routes/api/v1/favorites.js

```diff
@@ -54,5 +58,54 @@ router.post("/", function(req, res) {
     })
 
     .catch(error => res.send({error}));
-})
+});
+
+router.get("/", function(req, res) {
+  User.findOne({
+    where: {
+      api_key: req.body.api_key
+    },
+    include: [Location]
+  })
+    .then( user => {
+      if(!user){
+        res.sendStatus(401)
+      } else{
+        let favorites = [];
+        user.getLocations()
+          .then(locations => {
+            var userLocations = locations;
+            for (i = 0; i < locations.length; i++) {
+              let currentLocation = locations[i];
+              let forecastUrl = `https://api.darksky.net/forecast/${process.env.DARK_SKY_API_KEY}/${currentLocation.lat},${currentLocation.lng}?exclude=[minutely,alerts,flags]`;
+              fetch(forecastUrl)
```

Comment:
If you have time, you could think about breaking any fetch calls out into services, like we did with the original SweaterWeather, to clean this up a bit.